storybookjs/storybook
# Core: Add args reset API

Issue: #11487

## What I did

Add a new event/api `resetStoryArgs: (argName?: string) => void` which either:
 - resets a single arg's value OR
 - resets all args (if `argName` is unset).

@shilman  -- even though we don't need it, do you think instead this should be `resetStoryArgs: (argNames?: [string]) => void`?

## How to test

- Is this testable with Jest or Chromatic screenshots?
Yes see tests

- Does this need a new example in the kitchen sink apps?
Yes, see the `core/args.stories`

- Does this need an update to the documentation?
Maybe?
